tracing/tracing/src/main/java/io/helidon/tracing/Tag.java

```diff
@@ -43,6 +43,21 @@ public abstract class Tag<T> {
      * Status code that was returned.
      */
     public static final TagSource<Integer> HTTP_STATUS = new NumberTagSource<>("http.status_code");
+
+    /**
+     * Tag marking a Database type.
+     */
+    public static final TagSource<String> DB_TYPE = new StringTagSource("db.type");
+
+    /**
+     * Tag marking a Database statement.
+     */
+    public static final TagSource<String> DB_STATEMENT = new StringTagSource("db.statement");
+
+    /**
+     * Tag marking an Error.
+     */
+    public static final TagSource<Boolean> ERROR = new BooleanTagSource("error");
```

Comment:
This tag does not belong here and should not be used (see comment above)

Reply:
removed